### PR TITLE
Update node scripts to create members per team file

### PR DESCRIPTION
**Pull request by Mouhajer-CO**
### JIRA link

Resolves [NTRNL-114](https://technologyprogramme.atlassian.net/browse/NTRNL-114)

### Change description

Update node scripts to create members per team file. Unit tests will be covered on [NTRNL-115](https://technologyprogramme.atlassian.net/browse/NTRNL-115).

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -35,6 +36,26 @@ export const getGitOrgData = async (what: WhatEnum, org: string, page = 1): Prom
 
 // ************************************************************ //
 
+export const setTeamsData = async ( ): Promise<void> => {
+    try {
+        for (const team of ORG_DATA.teams) {
+            const members: string = team["members_url"];
```
Comment (Harrisman05) [inline src/utils/index.ts:42]: Suggestion, alternative method to extract the members url. `"members_url"` property value itself is difficult as it has `{/members}` at the end, which forces usage of `.slice()` with hard-coded value `9` to extract url. Instead, we can construct members url from the base `"url"` property.

```
const membersUrl: string = `${team["url"]}/members`;
const membersData = await fetch(membersUrl, HEADERS);
```

Thoughts on this approach?

File: src/config/index.ts [hunk 2]
```diff
@@ -18,6 +22,10 @@ export const STATE_DEPENDENCIES: StateDependencies = {};
 export const STATE_FILE_NAME = "state.json";
 export const STATE_FILE_PATH = `${REPOS_DIRECTORY_PATH}/${STATE_FILE_NAME}`;
 
+export const TEAMS_FILE_NAME = "teams.json";
+export const TEAMS_FILE_PATH = `${REPOS_DIRECTORY_PATH}/${TEAMS_FILE_NAME}`;
+export const TEAMS_DATA: TeamsData = {};
```
Comment (Harrisman05) [inline src/config/index.ts:27]: V minor semantics... move `TEAMS_DATA` to line 25 to maintain consistent format with `STATE_DEPENDENCIES`

File: src/utils/index.ts (hunk 1 above)
Comment (Mouhajer-CO) [reply src/utils/index.ts:42]: I like it, it is more clean, the variable name has to be changed as well, instead of `membersUrl` should be `teamUrl`.

**Review by Harrisman05 — APPROVED** [review]
Looks good

**Pull request merged**